content/applications/general/email_communication/faq.rst

```diff
@@ -1,173 +1,232 @@
+
+.. tip::
+   What is a **cron**? A cron is an action that Odoo runs in the background to execute particular
+   code to complete a task.
+
+.. important::
+   By default, the *Mass Mailing cron* runs every 60 minutes. This can be changed to no less than 5
+   minutes. However, running the action every 5 minutes would bog down the Odoo database (stress the
+   system), so this is not recommended. To edit the mass mailing cron, select the action
+   :guilabel:`Mail: Email Queue Manager`, and proceed to make any necessary adjustments.
 
-- The **Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
+Emails that are considered urgent (communication from one person to another, such as sales orders,
+invoices, purchase orders, etc.) are sent immediately.
 
-- The folder :file:`~/logs/` (preferably accessed by the command line) of an Odoo.sh contains
-  a list of files containing the logs of the database. The log files are created everyday
-  at 5:00 AM UTC. The two last days are not compressed, while the older ones are, in order
-  to gain space. The naming of the files for Today and Yesterday are :file:`odoo.log` and
-  :file:`odoo.log.1`. For the following, they are named with their dates and compressed.
-  See the Odoo.sh documentation about :ref:`logs <odoosh/logs>`. Use the command ``grep`` and
-  ``zgrep`` (for the compressed ones) to search through the files.
+Incoming emails
+===============
 
-- **Odoo Online** users won’t have access to their logs. However you can still contact
-  `Odoo Support <https://www.odoo.com/help>`_ , if you have a recurring issue
-  with the same client or domain.
+When there is an issue with incoming emails, there might not be an indication, per se, in Odoo. It
+is the sending email client, who tries to contact a database, that will get a bounce-back message
+(most of the time a :guilabel:`550: mailbox unavailable` error message).
 
-Get help from support
+Email is not received
 ---------------------
 
-In order to get helped efficiently, please provide as much information as possible. Here is a list
-of what can be helpful:
+The steps that should be taken depend on the Odoo platform where the database is hosted.
 
-- The **EML** of the email, stating for *Electronic Mail*, is the file format containing all the
-  technical information required for an investigation. The documentation of your own email provider
-  might help you on how to get your EML files. Once you get the EML of the email, adding it
-  in the attachment of your ticket is the most efficient way for us to investigate. The support
-  will mainly focus on redundant issues.
+**Odoo.sh** users can find their live logs on the folder :file:`~/logs/`.
 
-  .. seealso::
-     - `Gmail documentation
-       <https://support.google.com/mail/answer/29436>`_
-     - `Outlook documentation
-       <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
+.. tip::
```

Comment:
I really think that the first two paragraphs of this Tip should be normal text, not inside the Tip box